### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by 4812571**

### CODE REVIEW

File: .github/workflows/test_fuzz.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Mayhem
```
Comment (xansec) [inline .github/workflows/test_fuzz.yml]: Please rename this to `mayhem.yml`

File: mayhem/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Google LLC.
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Copyright and package stage.

File: mayhem/build.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
```
Comment (xansec) [inline mayhem/build.sh:1]: These commands can go in the Dockerfile

File: mayhem/test_fuzz.mayhemfile [hunk 4]
```diff
@@ -0,0 +1,4 @@
+project: nostril
```
Comment (xansec) [inline mayhem/test_fuzz.mayhemfile]: What is "test_fuzz"?

File: mayhem/test_fuzz.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
```
Comment (xansec) [inline mayhem/test_fuzz.py]: Cna we rename this to explain what's acutally being fuzzed? "test_fuzz" is not very descriptive.

**Review by xansec — CHANGES_REQUESTED** [review]
Please do not commit crash files, this is dangerous and needlessly exposes the repo to attack. If you believe you have found a legitimate issue, the proper thing to do is responisbly disclose following SECURITY.md reporting guidelines if they are there. Please make the requested changes.

**Review by xansec — COMMENTED** [review]
Also, let's remove the crash files as this is a security concern.

File: mayhem/test_fuzz.py (hunk 5 above)
Comment (4812571) [reply mayhem/test_fuzz.py]: I have renamed this to 'nonsense_fuzz' as it is fuzzing the 'nonsense' function of this library.

File: mayhem/test_fuzz.mayhemfile (hunk 4 above)
Comment (4812571) [reply mayhem/test_fuzz.mayhemfile]: I've renamed it as per your other suggestion

File: mayhem/build.sh (hunk 3 above)
Comment (4812571) [reply mayhem/build.sh:1]: I am using an OSS-Fuzz base for this submission, so I do not believe this is the case, or at least if it is, it is not conventional.

File: mayhem/Dockerfile (hunk 2 above)
Comment (4812571) [reply mayhem/Dockerfile:1]: I believe the copyright is necessary here.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**